clippy_lints/src/absolute_paths.rs

```diff
@@ -1,6 +1,7 @@
 use clippy_config::Conf;
 use clippy_utils::diagnostics::span_lint;
 use clippy_utils::is_from_proc_macro;
+use declare_clippy_lint::declare_clippy_lint;
```

Comment:
We could `#[macro_use]` the crate to not require lint modules to change, up to you

Reply:
I totally forgot that's an option. I personally prefer not having things magically added to a scope, but this is a case where it's not unreasonable.